### ISSUE TITLE
Warning: Zend OPcache API is restricted by "restrict_api" configuration directive

### ISSUE THREAD

**Issue by iandunn** [Rarst/laps#58]
I get this error when using the plugin on a server that has `opcache.restrict_api` configured.

> Warning: Zend OPcache API is restricted by "restrict_api" configuration directive in laps/src/Record/Collector/Core_Load_Collector.php on line 40

I think this line needs an additional check to make sure the plugin is within the restricted path, if one exists.

https://github.com/Rarst/laps/blob/752271b2588001d6ce344e8790838c1f96128f74/src/Record/Collector/Core_Load_Collector.php#L39-L40

**Comment by iandunn** [comment]
I applied that commit and it fixed the warning for me 👍🏻 

My understanding is that an `opcache.restrict` value of `/var/www/foo/` [should grant access for files in that directory, recursively](https://serverfault.com/questions/998586/enabling-opcache-restrict-api-for-security-reasons). Unfortunately I don't have time to dig deeper and test locally though.

**Comment by Rarst** [comment]
Yeah, it's borked for some combination of Apache and whatever https://github.com/php/php-src/issues/8799

I've threw in some silencing there, since I don't see a good way to make it behave if the upstream logic doesn't, and released in 3.3.7.

**Comment by iandunn** [comment]
Ah, good find. Thanks for the quick patch!
